client/src/components/FeedbackObject/index.js

```diff
@@ -68,49 +69,51 @@ const FeedbackObject = ({ feedbackToShow, student, updateFeedback }) => {
         <div className="feedback-title-and-module">
           <div className="feedback-title">
             <input
+              className="prev-feedback-title-input"
               name={"title"}
-              value={currentFeedback.title}
+              value={currentFeedback.title || ""}
               onChange={handleEdit}
               disabled={!isEditing}
             />
           </div>
           <div className="feedback-module">
-            <p className="feedback-input-heading">Module:</p>
             {!isEditing ? (
-              <p className="module-select-p">{currentFeedback.module}</p>
+              <p className="module-select-p">{`Module: ${currentFeedback.module}`}</p>
             ) : (
-              <select
-                className="module-select"
-                name={"module"}
-                value={currentFeedback.module}
-                onChange={handleEdit}
-              >
-                {modules.map((module, index) => (
-                  <option key={index} value={module.name}>
-                    {module.name}
-                  </option>
-                ))}
-              </select>
+              <>
+                <p className="feedback-input-heading">Module:</p>
+                <select
+                  className="module-select"
+                  name={"module"}
+                  value={currentFeedback.module || ""}
```

Comment:
I am happy with this, but why exactly are you adding || "" to all values?

Reply:
IT was complaining **about a component is changing an uncontrolled input of type text to be controlled. Input elements should not switch from uncontrolled to controlled (or vice versa). Decide between using a controlled or uncontrolled input element for the lifetime of the component.** . So, by passing an empty string as a value, we are basically bypassing the warning as the component is accepting complete and empty input value. By doing that we wont be changing the status it.